### DISCUSSION TITLE
What is the root password for a VM install of T-Pot?

### DISCUSSION THREAD

**Discussion by n1ghtwalk3r** [telekom-security/tpotce#899] [Q&A] [answered]
What is the root password for a VM install of T-Pot?

During the installed I was asked for a password for tsec and username/password for the web admin

I can log into T-Pot from the direct console or SSH via the tsec user, but I can't do anything that needs root privileges.

I want to use the dump_es.sh on my T-Pot in Digital Ocean and then use restore_es.sh on my VMware T-Pot but I can't run this script with out root access

Can you please assist or point me in the right direction?

**Comment by t3chn0m4g3** [comment, marked as answer]
`sudo su -`, same password as `tsec`

**Reply by n1ghtwalk3r** [reply]
Thanks that worked, I was trying sudo su root and that didn't work

**Reply by MolerRS** [reply]
> 谢谢，我正在尝试 sudo su root 但没有用

I also encountered the same problem when installing the latest version of T-Pot. Have you found a solution?

**Comment by baishakh22** [comment]
When you create the password for ```tsec``` is your login password. And the username is ```tsec```

Username: tsec
password: <what you put for tsec password>
